src/content/administration/dashboard.mdx

```diff
@@ -4,75 +4,125 @@ description: Admin Dashboard
 id: dashboard
 ---
 
+import Image from '@theme/Image';
 import TiersList from '@theme/TiersList';
 
 # Admin Dashboard  <TiersList tiers="Scale Enterprise Self-Hosted" />
 
-The Admin Dashboard provides a web UI for administering some features of Okteto.
+The Admin Dashboard provides a web UI to see various details of your Okteto instance in a centralized location and offers features for managing parts of the Okteto experience for your users.
 
 You can access the Admin Dashboard by clicking on the `Admin` icon on the left sidebar.
 
+<p align="center"><Image src="/docs/administration/images/admin_dashboard_tabs.png" alt="Admin dashboard tabs" width="900" /></p>
+
+This Admin view consists of the following views currently:
+
+- [General](#overview)
+- [Users](#users)
+- [Nodes](#nodes)
+- [Namespaces](#namespaces)
+- [Previews](#previews)
+- [Secrets](#secrets)
+- [Catalog](#catalog)
+- [Installation](#installation)
+
+We will cover these in more detail below.
+
 ## Accessing the dashboard
 
 Only accounts with the administrator role will be able to access the Administration Dashboard. The first user signing up into your Okteto instance will automatically become the initial administrator.
 
 You can promote other accounts to the administrator role in the `Users` section of the Admin Dashboard or by adding the `dev.okteto.com/super: "true"` label to the corresponding `serviceAccount` in Kubernetes.
 
-## Overview Section
+## Overview
```

Comment:
In the Admin UI is named "General". We should change it here or in the product, whatever makes more sense

Reply:
```suggestion
## General
```

Good catch! Thank you